### PR TITLE
Documentation for the reply-by-email to comment feature

### PR DESCRIPTION
**Pull request by namangupta01**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
OK! just re-mark it as `ready` when it's good to merge. Thanks!

**Comment by namangupta01** [comment]
Changes are done! Should we merge this pr after reply-by-email start working.

File: README.md [hunk 1]
```diff
@@ -80,12 +80,18 @@ Secure connection is needed for OAuth authentication etc.
...
 
+***
+
 ### Bundle exec
 
 For some, it will be necessary to prepend your gem-related commands with `bundle exec`, for example, `bundle exec passenger start`; adding `bundle exec` ensures you're using the version of passenger you just installed with Bundler. `bundle exec rake db: setup`, `bundle exec rake db: seed` are other examples of where this might be necessary.
 
 ***
 
+### Public Lab now supports reply by email to comment feature. For more details regarding it go to [Doc](https://github.com/publiclab/plots2/blob/master/doc/EMAIL.md)
```
Comment (jywarren) [inline README.md]: Maybe this should be

```
### Reply-by-email

Public Lab now...  

... go to the [email documentation](...)
```

Just for easier-to-read formatting?

Thanks!

File: doc/EMAIL.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## How to start reply by email to comment feature
+
+With the merge of [fad88f](https://github.com/publiclab/plots2/commit/fad88fd72c9d08f5591bf4c0cab402160eb9ea54) Public lab now supports Reply-by-email to comment features to the various nodes. For implementing this feature we have used `mailman` gem which is a microframework for processing incoming email. More details about this can be found at [mailman](https://github.com/mailman/mailman).
```
Comment (jywarren) [inline doc/EMAIL.md]: Let's link to the PR, not the commit? so discussion can be read too.

**Comment by jywarren** [comment]
Few small requests but this is looking great!!!

**Comment by jywarren** [comment]
Great work, Naman!

**Pull request merged**